openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,31 @@
+"""
+Models to support persistent tasks.
+"""
+
+from django.db import models
+
+from openedx.core.lib.properties import JSONProperty
+
+
+class FailedTask(models.Model):
+    """
+    Representation of tasks that have failed
+    """
+    task_name = models.CharField(max_length=255, db_index=True)
+    task_id = models.CharField(max_length=255)
+    argstring = models.TextField(blank=True)
+    kwargstring = models.TextField(blank=True)
+    exc = models.CharField(max_length=255)
```

Comment:
Maybe it would make sense to create the index(es) when we write the management command. Is there a downside to holding off?